cpp/include/cuml/linear_model/ols_mg.hpp

```diff
@@ -0,0 +1,116 @@
+/*
+* Copyright 1993-2019 NVIDIA Corporation.  All rights reserved.
```

Comment:
Change to 2020

Reply:
I changed these, as well as updated the copyright to the cuml one and not the cumlprims_mg one (I missed out on some). @dantegd had pointed out earlier to me that these are different licenses.

Reply:
good fix!